source/contracts/reporting/Market.sol

```diff
@@ -124,38 +134,54 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
         tentativeWinningPayoutDistributionHash = _reportingToken.getPayoutDistributionHash();
         designatedReportPayoutHash = tentativeWinningPayoutDistributionHash;
         reportingWindow.updateMarketPhase();
+        IReputationToken _reputationToken = reportingWindow.getReputationToken();
+        // The owner gets the no-show REP bond
+        _reputationToken.transfer(owner, _reputationToken.balanceOf(this));
+        // The owner gets the reporter gas costs
+        require(getOwner().call.value(reporterGasCostsFeeAttoeth)());
         return true;
     }
 
-    function disputeDesignatedReport() public triggersMigration returns (bool) {
+    function disputeDesignatedReport(uint256[] _payoutNumerators, uint256 _attotokens) public triggersMigration returns (bool) {
         require(getReportingState() == ReportingState.DESIGNATED_DISPUTE);
         designatedReporterDisputeBondToken = DisputeBondTokenFactory(controller.lookup("DisputeBondTokenFactory")).createDisputeBondToken(controller, this, msg.sender, Reporting.designatedReporterDisputeBondAmount(), tentativeWinningPayoutDistributionHash);
         reportingWindow.getReputationToken().trustedTransfer(msg.sender, designatedReporterDisputeBondToken, Reporting.designatedReporterDisputeBondAmount());
-        updateTentativeWinningPayoutDistributionHash(tentativeWinningPayoutDistributionHash);
+        if (_attotokens > 0) {
+            require(derivePayoutDistributionHash(_payoutNumerators) != tentativeWinningPayoutDistributionHash);
```

Comment:
🤔 While it would be silly, is there really any reason to disallow someone from disputing A and then buying up A to the point where it is winning again?  They can do that anyway by just bringing B up to above `0` and then staking a bunch on A as a separate transaction.  TL;DR: I am not sure this requirement actually adds value.